docs/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8"/>
+    <title>keyboard-only-outlines demo</title>
+</head>
+<body>
+    <button>A button</button>
+    <input type="text"/>
+    <input type="number"/>
+    <input type="password"/>
+    <textarea></textarea>
+    <input type="radio" name="radio"/>
+    <input type="radio" name="radio"/>
+    <input type="radio" name="radio"/>
+    <br>
+    <br>
+    <button id="toggle">Toggle keyboard-only-outlines</button> <p id="state"> Current state: on</p>
```

Comment:
Can we wrap `on` in a `<span id="state">on</span>` and just update to on/off instead of the whole sentence?